distribution/ddf-common/src/main/resources/etc/default.policy

```diff
@@ -0,0 +1,79 @@
+priority "grant";
+
+deny {
+  // Read, write to, or execute any file.
+  permission java.io.FilePermission "<<ALL FILES>>", "read, write, execute";
+
+  permission java.util.PropertyPermission "javax.net.ssl.*", "read, write";
+
+  // Change current security manager
+  permission java.lang.RuntimePermission "setSecurityManager";
+
+  // Modify application permissions at will.
+  permission java.security.SecurityPermission "setPolicy";
+
+  // Load classes into any protection domain
+  permission java.lang.RuntimePermission "createClassLoader";
+};
+
```

Comment:
❗️ The migration:import will need delete permissions and readlink under ddf.home. It will also need read permission outside ddf.home.

Reply:
We can't do read outside ddf.home. We should be able to add the others for that command, however.

Reply:
We need the import to be able to read outside of ddf.home to compute checksum for cases where the customer has referenced files (e.g. certs or otherwise outside of ddf.home)
@lessarderic How do we handle that?

Reply:
@paouelle, how does the code handle read failures when doing a checksum on a file during export and import? Do we just warn the user and move or fail the export/import?

Reply:
We do generate warnings if we cannot read it but that would still break the migration import/export functionality here.